kolibri_explore_plugin/assets/src/views/DiscoveryPage.vue

```diff
@@ -15,8 +16,8 @@
     components: { DiscoveryPageCollections, DiscoveryPageFullContent },
     computed: {
       showCollectionsPage() {
-        // FIXME read this from a setting.
-        return false;
+        // FIXME return false if "use EK USB" was selected in the onboarding.
```

Comment:
Either way we should create a Phabricator task or two so there's a roadmap to converging these things, but how about in the Android app we skip that onboarding step when `newCollectionsPageFeature` is enabled? That way we can call this bit in the explore plugin _done_. The way I see it is eventually we'll want to have a similar experience either way, mostly for our own sanity. (So, the app can ship with metadata based on the content we know is in Endless Key USBs but then disable the content management UI).

Reply:
I'll answer this in Phabricator as it involves design specs!